adapter/service/outlvlswitch/routing.go

```diff
@@ -32,6 +33,13 @@ func RouteService(serviceRegistry adapter.ServiceRegistry) []*router.Routing {
 	}
 }
 
+func RouteServiceLevelTransitioning(serviceRegistry adapter.ServiceRegistry) []*router.Routing {
```

Comment:
Can be merged with `RouteService`

Reply:
This is how we do it. A thing/device might not support it, but adapter should.

Reply:
Adapter may be used to support many things with different capabilities.

Reply:
You have a dedicated method "IsSupported" for this purpose.